src/main/java/com/sda/controller/CostumeController.java

```diff
@@ -19,6 +20,9 @@ public CostumeController(OrderService orderService) {
         this.orderService = orderService;
     }
 
+    @Autowired
+    private CostumeRepository costumeRepository;
```

Comment:
controller nie powinien mieć w sobie zależności do repository - jedynie do serwisów
zamiast tego, w OrderService powinna pojawić się metoda "przelotkowa" do repository. W ten sposób będziemy mogli korzystać tutaj tylko z klasy serwisowej, bez Repository

no i autowired na polach to zło, nie powinno się go tak używać

Reply:
resolved.